scripts/publish.sh

```diff
@@ -42,7 +42,8 @@ fi
 if [ "$PRODUCTION" = "1" ] && [ "$CHANNEL" = "dev" ]; then
     echo "Sync with ${GITHUB_REF} and push to it"
     git pull github "${GITHUB_REF}" --ff-only
-    git push github HEAD:"${GITHUB_REF}"
+    git tag -a "v$NEXT_EXTENSION_VERSION-dev" -m "v$NEXT_EXTENSION_VERSION-dev" -m "Prisma version: $PRISMA_VERSION"
```

Comment:
Just curious - in the past a tag didn't start with `v` instead just the version number. Is that supposed to be changed now?

Reply:
It is also inconsistent how we tag the unstable versions in prisma-engines: https://github.com/prisma/prisma-engines/releases

Reply:
Ah, that is a Github recommendation, will remove it for consistency.

![image](https://user-images.githubusercontent.com/746482/80566319-55187f00-8a10-11ea-9b6f-ab136e7f7fb4.png)

> It is also inconsistent how we tag the unstable versions in prisma-engines

Because the version of extension itself is inconsistent. I don't follow what you suggestion here is?

Reply:
Not to put a `v` in front of the version, different to how we tag all our other stuff.

Reply:
Removed `v` prefix